src/state/slices/marketDataSlice/selectors.ts

```diff
@@ -66,6 +66,20 @@ export const selectMarketDataById = createCachedSelector(
   },
 )((_state: ReduxState, assetId: AssetId | undefined): AssetId => assetId ?? 'undefined')
 
+export const selectMarketDataByIdInUSD = createCachedSelector(
```

Comment:
Should be `selectMarketDataFiatById` to honor our XyzFiat / XyzCrypto terminology. AFAIK USD default should be implied regardless of multi-currency settings with the processing from USD value to currency conversion happening further down the line

Reply:
this was the wrong approach, I fixed the bug which made numbers multiplied by rate^2, we no longer need this.